### ISSUE TITLE
Misleading error message when require doesn’t resolve

### ISSUE THREAD

**Issue by kleinfreund** [11ty/eleventy#182]
Requiring a package that is not yet installed in `.eleventy.js` like this:

```js
const CleanCSS = require("clean-css");
```
plus having a shortcode defined like this:

```js
eleventyConfig.addLiquidShortcode('excerpt', post => extractExcerpt(post));
```

produces the following incorrect error when running `eleventy`:

```
Problem writing eleventy templates (more info in DEBUG output): 
 * Error: Having trouble rendering template ./_includes/layouts/post-list.liquid: <div...
(file content stripped)
 * ParseError: tag excerpt not found, line:7, line:2
```

Removing the `require` or doing a `npm install --save-dev clean-css` returns everything to a working build.

**Comment by zachleat** [comment]
As of 0.5.4:

```
// const CleanCSS = require("clean-css");

module.exports = function(eleventyConfig) {
	eleventyConfig.addFilter("cssmin", function(code) {
		return new CleanCSS({}).minify(code).styles;
	});
};
```

![image](https://user-images.githubusercontent.com/39355/45425926-512fc180-b660-11e8-8d27-0e08f8e5782e.png)

```
// const CleanCSS = require("clean-css");

module.exports = function(eleventyConfig) {
	eleventyConfig.addFilter("cssmin", function(code) {
		return new CleanCSS({}).minify(code).styles;
	});
};
```

## Here’s the issue:

```
const CleanCSS = require("this-is-a-made-up-module-name");

module.exports = function(eleventyConfig) {
	eleventyConfig.addFilter("cssmin", function(code) {
		return new CleanCSS({}).minify(code).styles;
	});
};
```

gave me some unrelated error:
![image](https://user-images.githubusercontent.com/39355/45426088-d4511780-b660-11e8-91ce-ce8a4eed6906.png)

**Comment by zachleat** [comment]
![image](https://user-images.githubusercontent.com/39355/45426207-2e51dd00-b661-11e8-8a65-3912f1868a5c.png)

should throw an error:
![image](https://user-images.githubusercontent.com/39355/45426213-34e05480-b661-11e8-86bb-aed2f3c799d4.png)

**Comment by zachleat** [comment]
Fixed new behavior in 0.5.4:

![image](https://user-images.githubusercontent.com/39355/45427143-c224a880-b663-11e8-9058-29354bfd820c.png)
